### DISCUSSION TITLE
Issue related to kibana docker container

### DISCUSSION THREAD

**Discussion by solftclone** [Security-Onion-Solutions/securityonion#9584] [Unsupported Versions]
Hi, I am using security onion 170 version.there kibana 8.4.1 version is available. I want to use wazuh kibana app plugin but for kibana version 8.4.1 there is no wazuh kibana app plugin available. Hence I was thinking to have two kibana docker containers running on my security onion and install wazuh kibana app on another kibana container like 7.16.5. can you please guide me on is it possible and how can I do that

**Comment by cm-ops** [comment]
We don't currently support the Wazuh App for Kibana.
